src/main/webapp/script.js

```diff
@@ -49,7 +49,7 @@ async function getGuide() {
 }
 
 function guidesToHTML(guide) {
-    return `<div class="card w-75 itemBox item1">
+    return `<div class="card w-75 itemBox backend">
```

Comment:
What does adding the `backend` class do?

Reply:
So currently, it's only filtering the backend when clicked from the navigation. It basically hardcoded. I'll be replacing `backend` with `${guide.category}`